pkg/apis/servicecatalog/v1beta1/plan.go

```diff
@@ -65,3 +95,33 @@ func (p *ClusterServicePlan) GetClassID() string {
 func (p *ServicePlan) GetClassID() string {
 	return p.Spec.ServiceClassRef.Name
 }
+
+// GetInstanceCreateSchema returns the instance create schema from plan.
```

Comment:
This is a FYI to other reviewers, that we know that this is a lot of wrapper methods here. We are working on improving how we can work with cluster scoped and namespaced scoped specs without as much effort in #2379 .